install/jaxws/docs/assertions/JAXWS30SpecAssertions.html

```diff
@@ -72,7 +72,7 @@ <h2><br>
 </TR>
 <TR>
 <TD align="center" valign="center"><a name="JAXWS:SPEC:2075"></a><font size="1PT">JAXWS:SPEC:2075</font></TD><TD align="center" valign="center"><font size="1PT">2</font></TD><TD align="center" valign="center"><font size="1PT"></font></TD><TD align="left" valign="center"><font size="1PT">
-                        Conformance (WSDL Addressing Support): An implementation MUST support the mapping of WS-Addressing 1.0 WSDL Metadata[32] to Java.  
+                        Conformance (WSDL Addressing Support): An implementation MUST support the mapping of WS-Addressing 1.0 WSDL Metadata[32] to Java.
```

Comment:
@Tomas-Kraus did you intend to delete all of the trailing spaces?  That makes the pull request longer to review.

Reply:
I'm using Atom to edit files and it removed trailing spaces automatically.